hey/patterns/comments.php

```diff
@@ -14,7 +14,7 @@
 		<div class="wp-block-group" style="margin-top:0;margin-bottom:var(--wp--preset--spacing--50)">
 			<!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap"},"style":{"spacing":{"blockGap":"0.5em"}}} -->
 			<div class="wp-block-group">
-				<!-- wp:avatar {"size":40,"style":{"border":{"radius":"999px"}}} /-->
+				<!-- wp:avatar {"size":40} /-->
```

Comment:
Why did we need to remove this circular avatar style?

Reply:
> Keeping it here is not maintainable.

I don't understand. This wasn't added css to enhance the block. The border radius setting is already in the editor, and I just used it in the same way as the size of the avatar.

![Screenshot 2023-03-27 at 11 27 30](https://user-images.githubusercontent.com/908665/227917090-398ad20a-01d5-4708-a26c-d47d3a4f9541.jpg)

Reply:
That sounds good. Thank you!